visualizations/frontend/filters/TTagsSelector.vue

```diff
@@ -321,6 +313,29 @@ export default {
       }
       return (this.expanded = this.expanded.filter((k) => k !== key));
     },
+    searchInObject(searchTerm, obj) {
+      const lowercaseSearchFields = this.searchFields
+        ? this.searchFields.map((field) => field.toLowerCase())
+        : null;
+      for (const key of Object.keys(obj)) {
+        if (
+          !this.searchFields ||
+          lowercaseSearchFields.includes(key.toLowerCase()) ||
+          key.toLowerCase() === "value"
+        ) {
+          const val = obj[key];
+          if (
+            val?.value
+              ?.toString()
+              .toLowerCase()
+              .includes(searchTerm.toLowerCase())
+          ) {
+            return true;
+          }
+        }
+      }
+      return false;
+    },
```

Comment:
```suggestion
    searchInObject(searchTerm, row) {
      const COLUMN_VALUE = "value";
      const lowercaseSearchFields = this.searchFields
        ? this.searchFields.map((field) => field.toLowerCase())
        : [];
        
      for (const column of Object.keys(row)) {
        if (
          !this.searchFields ||
          lowercaseSearchFields.includes(column.toLowerCase()) ||
          column.toLowerCase() === COLUMN_VALUE
        ) {
          const data = row[column];
          
          if (
            data
              ?.value
              ?.toString()
              .toLowerCase()
              .includes(searchTerm.toLowerCase())
          ) {
            return true;
          }
        }
      }
      return false;
    },
```

Reply:
I have not tested the suggestion locally, it just a semantic change not logic change involved.